test/e2e/framework/framework.go

```diff
@@ -443,7 +443,7 @@ func (f *Framework) WriteFileViaContainer(podName, containerName string, path st
 			return fmt.Errorf("Unsupported character in string to write: %v", c)
 		}
 	}
-	command := fmt.Sprintf("echo '%s' > '%s'", contents, path)
+	command := fmt.Sprintf("echo '%s' > '%s';sync;sync", contents, path)
```

Comment:
Doesn't the quoted article say that `sync; sync` is actually a waste of time and not something that competent unix people should do anymore?